opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -438,6 +438,9 @@ def use_span(self, span: "Span") -> typing.Iterator[None]:
 
         Args:
             span: The span to start and make current.
+
+        Yields:
+            `None`
```

Comment:
Backticks here because this is interpreted as a text description of the thing that we yield. `None: some description` would force "None" to render as a type.

Ideally we'd pick up the yielded type from the return type annotation, but `sphinx_autodoc_typehints` isn't smart enough to unpack `-> typing.Iterator[None]` above.